packages/api-client-core/src/operationRunners.ts

```diff
@@ -302,21 +307,69 @@ export const actionRunner: ActionRunner = async <Shape extends RecordShape = any
   }
 };
 
+const processBulkActionResponse = <Shape extends RecordShape = any>(
+  defaultSelection: FieldSelection | null,
+  response: any,
+  records: any,
+  modelSelectionField: string,
+  hasReturnType?: HasReturnType | null
+) => {
+  if (defaultSelection == null) return [];
+  if (!hasReturnType) {
+    return hydrateRecordArray<Shape>(response, records[modelSelectionField]);
+  } else if (typeof hasReturnType == "boolean") {
+    return records.results;
```

Comment:
If `hasReturnType === false`, should we also be hydrating?

Reply:
yup; that's covered above by `if (!hasReturnType)`

Reply:
Oh right! 🤦